### PR TITLE
Python to firebase crashlytics

### PR DESCRIPTION
**Pull request by xcho1**

### CODE REVIEW

File: src/java/org/learningequality/FullScreen.java [hunk 1]
```diff
@@ -51,21 +42,6 @@ public void onPageFinished(WebView view, String url) {
...
-                    button.setLayoutParams(new LinearLayout.LayoutParams(
-                            LinearLayout.LayoutParams.MATCH_PARENT,
-                            LinearLayout.LayoutParams.MATCH_PARENT));
-
-                    viewGroup.addView(button);
-
-                    button.setOnClickListener(v -> {
-                        throw new RuntimeException("This is a test crash");
-                    });
-                }
```
Comment (dylanmccall) [inline src/java/org/learningequality/FullScreen.java:68]: Just to make things tidier, We should do a rebase so this change is in its own commit. Best thing might be to remove the change from this commit and instead add `git revert 150edf919d92b7cf953e895bdb645eaeb7bcd19e`.

**Review by dylanmccall — CHANGES_REQUESTED** [review]
A small commit history nitpick, but the final change set looks great!

**Comment by dylanmccall** [comment]
Okay, I went ahead and did that rebasing! Need another reviewer, but I'm happy with the result now. I'm really looking forward to using this :)

**Comment by xcho1** [comment]
@dylanmccall, yeah it makes sense to split them into separate commits, thanks!

**Comment by dylanmccall** [comment]
I gave this another look today and it's working for me, so I'll go ahead and merge it. Here's an example of a Python exception I was able to produce by abusing the `options.ini` file:

https://console.firebase.google.com/project/endless-key/crashlytics/app/android:org.endlessos.Key/issues/42f106d556a9a23ac402ae39064e4a3d?time=last-hour&sessionEventKey=639272D502B900011DB8235F4C39AC15_1753369069143524891

We might like to add more data to these crashes in the future as our needs evolve, but it seems pretty clear how to do that.

**Comment by manuq** [comment]
Nice! The report is a bit hard to read but it is something!

**Comment by dylanmccall** [comment]
> Nice! The report is a bit hard to read but it is something!

One thing that helps here is switching to plain text instead of formatted text in the stack trace viewer, so the line breaks work. Definitely a little crude, though.

**Comment by manuq** [comment]
> > Nice! The report is a bit hard to read but it is something!
> 
> One thing that helps here is switching to plain text instead of formatted text in the stack trace viewer, so the line breaks work. Definitely a little crude, though.

Ah good advice thanks!

**Pull request merged**